clang/include/clang/Basic/AttrDocs.td

```diff
@@ -7401,7 +7495,8 @@ b for constant buffer views (CBV).
 
 Register space is specified in the format ``space[number]`` and defaults to ``space0`` if omitted.
 Here're resource binding examples with and without space:
-.. code-block:: c++
+
+.. code-block:: hlsl
```

Comment:
I think the two code blocks you changed here to HLSL are fine, but just beware that we do sometimes intentionally use the C++ syntax highlighting because the HLSL highlighting isn't always as good.

Reply:
In these two cases it is either identical or an improvement see examples:

Before my change:
![image](https://github.com/llvm/llvm-project/assets/1802579/c24c6009-ddec-4d89-83a9-25cca05a9846)
![image](https://github.com/llvm/llvm-project/assets/1802579/eb2d8414-a5a4-46d9-8e73-812d4fdbcf98)

After my change:
![image](https://github.com/llvm/llvm-project/assets/1802579/c3af0b29-e672-4b88-a394-25032c3db3fa)
![image](https://github.com/llvm/llvm-project/assets/1802579/fd2debd8-d554-4e76-9b1a-4d393b3c6581)